app/authentication/permissions.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsAdminUser(BasePermission):
```

Comment:
You do not need to write this yourself. Django Rest Framework has it built in! https://www.django-rest-framework.org/api-guide/permissions/#isadminuser